gui.py

```diff
@@ -0,0 +1,99 @@
+from conversion_manager import ConversionManager
+from path_helper import PathHelper
+from PyQt6.QtCore import QSize
+from PyQt6.QtGui import QIcon
+from PyQt6.QtWidgets import QApplication, QMainWindow, QPushButton, QVBoxLayout, QHBoxLayout, QLabel, \
+    QLineEdit, QWidget, QFileDialog, QMessageBox
+
+
+class MainWindow(QMainWindow):
+    def __init__(self, conversion_manager: ConversionManager, input_path: str, output_path: str) -> None:
+        super().__init__()
+        self.setMinimumSize(QSize(450, 150))
+
+        self.conversion_manager: ConversionManager = conversion_manager
+
+        self.setWindowTitle("Аршин-конвертер")
+        self.layout = QVBoxLayout()
+        input_layout = QHBoxLayout()
+        output_layout = QHBoxLayout()
+
+        input_label = QLabel('Выбрать файл')
+        input_label.setMinimumSize(QSize(85, 15))
+        self.input_line_edit = QLineEdit()
+        search_input_button = QPushButton("Обзор")
```

Comment:
You can probably call it something like `open_button` or `button_open`. What comes first is for you to decide, but if you change that, you should make other control names consistent with that.

Sometimes it's useful to have all buttons be called `button_*` and text inputs like `text_*` or something like that. But sometimes it's not.